jep/234/README.adoc

```diff
@@ -0,0 +1,274 @@
+= JEP-0000: Customizable Jenkins header
```

Comment:
needs to be in a file named `jep/0000/README.adoc`

Reply:
Updated in https://github.com/jenkinsci/jep/pull/380/commits/e1d733370cf1bdba8d4591b9190f55999e5e5370